src/main/java/org/jabref/gui/preferences/external/ExternalTabViewModel.java

```diff
@@ -81,7 +83,8 @@ public ExternalTabViewModel(DialogService dialogService, PreferencesService pref
     }
 
     public void setValues() {
-        eMailReferenceSubjectProperty.setValue(initialPreferences.getEmailSubject());
+        defaultEMailReferenceSubject = Localization.lang((String) preferences.getDefaults().get(JabRefPreferences.EMAIL_SUBJECT));
```

Comment:
As we already discussed in the DevCall, I'd say: Thou shall not access low level code directly from high level context. ;-)

Reply:
So what do you propose we do instead? If we cannot access the translated default here, this approach won't work.

Reply:
Im thinking of completly different approach to defining defaults. Instead of working in the low level api of JabRefPreferences with the internal map strings, the defautls could be placed in another class `DefaultPreferences` which implements PreferencesServices.

Reply:
Maybe this could be a solution for our localized preferences debate?